.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "env": {
+    "browser": true
+  },
+  "extends": [
+    "airbnb",
+    "prettier"
+  ],
+  "plugins": [
+    "prettier"
+  ],
+  "rules": {
+    "prettier/prettier": "error"
```

Comment:
Maybe better replace with:
```
    "prettier/prettier": [
      "error",
      { "singleQuote": true, "trailingComma": "es5" }
    ]
```

Reply:
I'd like to keep defaults as much as possible so I'll leave the the quote thing. But the argument reagrding the diffs is valid so I will change it to `{ "tralingComma": "es5" }`